recipes/apsg/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "apsg" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 64f471819be605425e43608e094713c29d072b724683baa41bfd4e98af698688
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vv"
+
```

Comment:
you can also add `entry_points` here as per https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#python-entry-points

Reply:
I see you have https://github.com/ondrolexa/apsg/blob/master/setup.py#L33

Reply:
Thanks, entry points used in setup.py added to recipe.